src/engine/strat_engine/tests/util.rs

```diff
@@ -0,0 +1,61 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+use std::path::PathBuf;
+
+use devicemapper::{DM, DmFlags, DevId};
+
+use nix::mount::umount;
+use mnt::get_submounts;
+
+/// Remove all device mapper devices which have a name which matches the naming convention for
+/// a stratis created device.
+fn dm_stratis_devices_remove() {
+
+    let dm = DM::new().unwrap();
+
+    // Lets assume that nesting won't go more than 10 deep...
```

Comment:
You could drop this comment and put the truer facts in the header. Something like:

```
/// iteratively remove device mapper devices which have a name which matches the naming convention for a stratis-created device.
/// This method does at most ten passes, removing all devices which are not depended on by another device at each pass.
```

Reply:
IMHO the header documentation should state what it does, not how it does it.  When we get a fix for the kernel oops bug, then there will be no more for loop, just a single pass across the devices that match.

Reply:
I agree with the sentiment. But unfortunately the header comment says cleans up _all_ devices and "cleans up some devices" seems a bit slapdash.

What about keep cleans up all, with a FIXME indicating that it will really and truly clean up all when it is able to call ```DM::remove_all()``` ?